compose_files/sql/users.sql

```diff
@@ -1,22 +1,148 @@
-set define on
+   set define on
 define OFFICE_EROC=&1
+defin API_KEY=&2
 begin
-    cwms_sec.add_user_to_group('&&OFFICE_EROC.webtest','All Users', 'HQ');
-    cwms_sec.add_user_to_group('&&OFFICE_EROC.webtest','All Users', 'SPK');
-    cwms_sec.add_user_to_group('&&OFFICE_EROC.webtest','CWMS Users', 'HQ');
-    cwms_sec.add_user_to_group('&&OFFICE_EROC.webtest','CWMS User Admins', 'HQ');
-    
-
-    cwms_sec.add_cwms_user('l2hectest',NULL,'SPK');
-    cwms_sec.update_edipi('l2hectest',1234567890);
-    cwms_sec.add_user_to_group('l2hectest','All Users', 'SPK');
-    cwms_sec.add_user_to_group('l2hectest','CWMS Users', 'SPK');
-    cwms_sec.add_user_to_group('l2hectest','TS ID Creator','SPK');
-
-    cwms_sec.add_cwms_user('l1hectest',NULL,'SPL');
+   cwms_sec.add_user_to_group(
```

Comment:
the formatting here doesn't need to be changed, and this change makes it more difficult to see at a glance what is setup.